mne/viz/_3d.py

```diff
@@ -415,6 +453,20 @@ def plot_source_estimates(stc, subject=None, surface='inflated', hemi='lh',
         View to use. See surfer.Brain().
     colorbar : bool
         If True, display colorbar on scene.
+    clim : str | dict
+        Colorbar properties specification. If 'auto', set clim automatically
+        based on data percentiles. If dict, should contain:
+            kind : str
+                Flag to specify type of limits. 'value' or 'percent'.
+            lims : length 3 list, array, or tuple
+                Note: Only use this if 'colormap' is not 'mne_analyze'.
+                Left, middle, and right bound for colormap.
+            pos_lims : length 3 list, array, or tuple
+                Note: Only use this if 'colormap' is 'mne_analyze'.
+                Left, middle, and right bound for colormap. Positive values
+                will be mirrored directly across zero during colormap
+                construction to obtain negative control points.
```

Comment:
oops, I think I was responding to some other thread. If we use `pos_limits`, isn't it mirrored so that you have 6 points / bounds (not sure about the exact terminology)?